apps/trustlab/src/payload/fields/colorPicker.js

```diff
@@ -0,0 +1,35 @@
+import { deepmerge } from "@mui/utils";
+import { ColourTextField } from "@nouance/payload-better-fields-plugin/ColourText";
+import { validateHTMLColorHex } from "validate-color";
+
+const colorPicker = ({ backgroundOverrides = {}, textOverrides = {} }) => {
+  const background = deepmerge(
+    {
+      name: "backgroundColor",
+      admin: {
+        description: "Background color in hex format",
+      },
+      required: true,
+      validate: (value) => validateHTMLColorHex(value) || "Invalid hex color",
```

Comment:
You can create a single const `validateColor = (value) => validateHTMLColorHex(value) || "Invalid hex color";` function and reuse it rather than duplicate the code.